### DISCUSSION TITLE
Add option to add a skin to a mod preset

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#21931] [Ideas] [answered]
With the ability to create mod presets, some people will use it to use mods they use with a specific skin (For example, Black and white skins with DT), so perhaps adding the option to add a skin to a mod preset could be useful

**Comment by peppy** [comment, marked as answer]
This is out of scope. We will be adding game-wide settings profiles though, which could be used to save skin / settings presets.
